gcp_airflow_foundations/source_class/ftp_source.py

```diff
@@ -29,39 +33,64 @@ def get_bq_ingestion_task(self, dag, table_config):
         taskgroup = TaskGroup(group_id="ftp_taskgroup")
 
         metadata_sensor_task = self.metadata_file_sensor(table_config, taskgroup)
```

Comment:
do we need to create all 3 tasks every time? (metaadata, flag, schema) or can we create only one based on the config? 
creating extra dummy tasks makes the DAg look ugly, and effects scheduler performance

Reply:
yeah, mentioned in the PR this is something I want to do. I think i will add this functionality today